kubernetes/kfserving/Huggingface_readme.md

```diff
@@ -14,17 +14,23 @@ This produces all the required files for packaging using a huggingface transform
 The .mar file creation command is as below:
 
 ```
-torch-model-archiver --model-name BERTSeqClassification --version 1.0 --serialized-file serve/examples/Huggingface_Transformers/Transformer_model/pytorch_model.bin --handler serve/examples/Huggingface_Transformers/Transformer_model/Transformer_handler_generalized.py --source-vocab serve/examples/Huggingface_Transformers/Transformer_model/vocab.txt --extra-files "Transformer_model/config.json,serve/examples/Huggingface_Transformers/Transformer_model/setup_config.json,serve/examples/Huggingface_Transformers/Transformer_model/index_to_name.json"
+torch-model-archiver --model-name BERTSeqClassification --version 1.0 --serialized-file serve/examples/Huggingface_Transformers/Transformer_model/pytorch_model.bin --handler serve/examples/Huggingface_Transformers/Transformer_model/Transformer_handler_generalized.py --extra-files "serve/examples/Huggingface_Transformers/Transformer_model/vocab.txt,serve/examples/Huggingface_Transformers/Transformer_model/config.json,serve/examples/Huggingface_Transformers/Transformer_model/setup_config.json,serve/examples/Huggingface_Transformers/Transformer_model/index_to_name.json"
 ```
 
-## Starting Torchserve
+## Starting Torchserve for KFServing Predictor
 To serve an Inference Request for Torchserve using the KFServing Spec, follow the below:
 
 * create a config.properties file and specify the details as shown:
 ```
+inference_address=http://127.0.0.0:8085
+management_address=http://127.0.0.0:8081
+number_of_netty_threads=4
 service_envelope=kfserving
+job_queue_size=10
+model_store=model-store
+
 ```
-The Service Envelope field is mandatory for Torchserve to process the KFServing Input Request Format.
+The Service Envelope field should be set as kfserving and it is mandatory.
```

Comment:
Please change to "The service_envelope=kfserving setting is needed when deploying models on KFServing"

Reply:
@chauhang  Changed them in the respective places